### PR TITLE
chore: remove prediction URL from output

### PR DESCRIPTION
**Pull request by balajismaniam**
For now, I've commented it. This was also tested using a local SiC deployment and is working as expected.

### CODE REVIEW

**Comment by balajismaniam** [comment]
Some context: The URL for Prediction Request is still part of SiC config. However, the easiest way to make sure this doesn't show up in the deployment is to remove it from the code repo. The SiC just skips it.

File: infra/metadata.display.yaml [hunk 1]
```diff
@@ -16,8 +16,6 @@ apiVersion: blueprints.cloud.google.com/v1alpha1
 kind: BlueprintMetadata
 metadata:
   name: terraform-ml-image-annotation-gcf-display
-  annotations:
```
Comment (ivanmkc) [inline infra/metadata.display.yaml:19]: I'm not sure what this does.

File: infra/modules/storage/metadata.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ spec:
     source:
       repo: https://github.com/GoogleCloudPlatform/terraform-ml-image-annotation-gcf.git
       sourceType: git
-      dir: /infra/modules/storage
+      dir: storage
```
Comment (ivanmkc) [inline infra/modules/storage/metadata.yaml:27]: Is "storage" correct?

If it's relevant to the repo root, then "/infra/modules/storage" looks more correct.

File: infra/modules/cloudfunctions/metadata.yaml [hunk 3]
```diff
@@ -24,7 +24,7 @@ spec:
     source:
       repo: https://github.com/GoogleCloudPlatform/terraform-ml-image-annotation-gcf.git
       sourceType: git
-      dir: /infra/modules/cloudfunctions
+      dir: cloudfunctions
```
Comment (ivanmkc) [inline infra/modules/cloudfunctions/metadata.yaml:27]: If relative to repo root, shouldn't "/infra/modules/cloudfunctions" be correct?

**Review by ivanmkc — COMMENTED** [review]
Left some comments.

**Review by ivanmkc — APPROVED** [review]
LGTM

File: infra/metadata.display.yaml (hunk 1 above)
Comment (balajismaniam) [reply infra/metadata.display.yaml:19]: All changes to the metadata.yaml and metadata.display.yaml were auto-generated when I ran `make docker_generate_docs`. ATM these files are not used anywhere. But we are working with the blueprints team to improve the process. So these changes can be ignored.

File: infra/modules/storage/metadata.yaml (hunk 2 above)
Comment (balajismaniam) [reply infra/modules/storage/metadata.yaml:27]: All changes to the metadata.yaml and metadata.display.yaml were auto-generated when I ran `make docker_generate_docs`. ATM these files are not used anywhere. But we are working with the blueprints team to improve the process. So these changes can be ignored.

File: infra/modules/cloudfunctions/metadata.yaml (hunk 3 above)
Comment (balajismaniam) [reply infra/modules/cloudfunctions/metadata.yaml:27]: All changes to the metadata.yaml and metadata.display.yaml were auto-generated when I ran `make docker_generate_docs`. ATM these files are not used anywhere. But we are working with the blueprints team to improve the process. So these changes can be ignored.

**Comment by ivanmkc** [comment]
I think you might have to remove this line: https://github.com/GoogleCloudPlatform/terraform-ml-image-annotation-gcf/blob/c744eee3001a26b0bdcb0df3e432793d22331bcd/infra/test/integration/simple_example/simple_example_test.go#L49

**Comment by balajismaniam** [comment]
We can't remove this line. The test depends on the `/annotate` endpoint. I was planning to rename the variable.

**Comment by balajismaniam** [comment]
Closing this PR as we decided to remove from SiC config.

**Pull request closed without merging**